how-to/sharing_docs.md

```diff
@@ -0,0 +1,22 @@
+# Sharing Docs with the Community
+
+## Prerequisites
+
+You should be a member of the [kubeflow-discuss](https://groups.google.com/forum/#!forum/kubeflow-discuss) group in order to view docs in the shared Google Drive.
+
+## Moving Google Docs into the shared Google Drive
+
+1. Assign ownership of the Google doc to drive-content-managers@kubeflow.org ([instruction](https://support.google.com/drive/answer/2494892)).
+1. Open a GitHub issue to transfer the doc.
+1. Assign the issue to someone in the [content manager list](#shared-google-drive-content-managers).
+1. Content manager will then move the doc into the drive.
+
+## Shared Google Drive Content Managers:
```

Comment:
This is the list of people with @google.com email in the drive-content-managers file in https://github.com/kubeflow/internal-acls. I'm not able to find the GitHub usernames for those with only @kubeflow.org email.